### ISSUE TITLE
"Error parsing text-format caffe.NetParameter: 48:12: Message type "caffe.LayerParameter" has no field named "bn_param"

### ISSUE THREAD

**Issue by nurvarli** [xipeng13/recurrent-face-alignment#7]
Hi,
When I run this code, I get some error like below:

Error parsing text-format caffe.NetParameter: 48:12: Message type "caffe.LayerParameter" has no field named "bn_param".

Could you help me?

**Comment by ChiefGodMan** [comment]
Have you solved this problem?

**Comment by vkasojhaa** [comment]
This is because you are not using the caffe version that support batch normalization layer (such as [SegNet](https://github.com/alexgkendall/caffe-segnet)). I resolved the issue by using the caffe segnet version.

**Comment by nurvarli** [comment]
Yes, I solved this problem with caffe Segnet, too. This algorithm can find 7 landmarks default on test images. I have changed NLMK parameter with 68 but it did not work. I need 68 landmarks. Do you know, how is it possible?

**Comment by vkasojhaa** [comment]
I also need the pre-trained weights for 68 landmarks. I have opened an issue for the same. I guess you need to train the model on 68 landmarks but there is no code available to train the model.
